terraform/deployments/cluster-services/logging.tf

```diff
@@ -16,6 +16,20 @@ resource "helm_release" "filebeat" {
   timeout          = var.helm_timeout_seconds
   values = [yamlencode({
     daemonset = {
+      securityContext = {
+        allowPrivilegeEscalation = false
+        privileged               = false
+        runAsUser                = 1000
+        runAsNonRoot             = true
+        runAsGroup               = 1000
```

Comment:
Probably add `fsGroup: 1000` to ensure any directories and files are owned by the same user you're running as